templates/taxi/car_detail.html

```diff
@@ -9,4 +10,6 @@ <h1>Drivers</h1>
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
   {% endfor %}
   </ul>
+  <a href="{% url "taxi:car-update" pk=car.id %}" class="btn btn-primary">Update</a>
+  <a href="{% url "taxi:car-delete" pk=car.id %}" class="btn btn-danger">Delete</a>
```

Comment:
Check correctnes of using quoutes because you use "" and inside "" again

Reply:
Yep, sure, my bad, i forgot about that here